pom.xml

```diff
@@ -79,8 +82,9 @@
                     </argLines>
                     <runtimeConfiguration>
                         <discoverRuntimes>
-                            <product>ALL</product>
-                            <includeSnapshots>true</includeSnapshots>
+                            <product>EE</product>
```

Comment:
Why are we setting the enterprise editions?

Reply:
It's ok. I was just wondering why we were testing with EE when a lot of connectors should work with the CE version.